Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -171,7 +171,7 @@ def __init__(self, tenant_id, auth_id, enc_key, app_name, base_url, verify, prox
         self.alert_status_to_fetch = alert_status_to_fetch
         self.alert_time_to_fetch = alert_time_to_fetch
         # TODO: Replace with v1 endpoint when out.
-        self.indicators_endpoint = 'https://graph.microsoft.com/beta/security/tiIndicators'
+        self.indicators_endpoint = 'https://graph.microsoft.com/beta/security/tiIndicators?$top=200&count=true'
```

Comment:
the top param should not be hard coded, but the limit command arg
also, why did you set `count=true`? i dont see you use the `@odata.count` property